karpenter.tf

```diff
@@ -92,6 +97,7 @@ spec:
   - id: ${aws_subnet.public[1].id}
   amiSelectorTerms:
     - name: "amazon-eks-node-${var.cluster_version}-*"
```

Comment:
This should be removed I think as it causes it to fall back to using the name as the selector and not the ID.